backend/seed_data/quiz_questions.yaml

```
@@ -466,3 +466,476 @@ quiz_questions:
         is_correct: false
         option_type: CONCEPT
 
+# Binary Trees Concept Quiz
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)